ndscheduler/corescheduler/core/base.py

```diff
@@ -14,6 +14,8 @@ class BaseScheduler (apscheduler_tornado.TornadoScheduler):
 
     # Seconds to wake up to see if it's okay to run.
     DEFAULT_WAIT_SECONDS = 60
+    # Number of args in self.run_job
+    RUN_JOB_ARGS = 5
```

Comment:
Could you also update https://github.com/Nextdoor/ndscheduler/blob/master/ndscheduler/corescheduler/utils.py#L50 to reference this as well

Reply:
Done. I moved the constant to the constants file as well.